packages/page-accounts/src/Sidebar/AddressSection.tsx

```diff
@@ -46,15 +50,36 @@ function AddressSection ({ accountIndex, defaultValue, editingName, flags, onCha
           value={value}
           withSidebar={false}
         />
+
         <div className='ui--AddressMenu-addr'>
           {value}
         </div>
+
         {accountIndex && (
           <div className='ui--AddressMenu-index'>
             <label>{t<string>('index')}:</label> {accountIndex}
           </div>
         )}
       </div>
+
+      <div className='ui--AddressSection__CopyColumn'>
+        <div className='ui--AddressMenu-copyaddr'>
+          <CopyToClipboard
+            text={value}
+          >
+            <span>
+              <Button.Group>
+                <Button
+                  icon={isCopyShown ? 'check' : 'copy'}
+                  label={wasCopied ? t<string>('Copy') : t<string>('Copy')}
```

Comment:
I think it's better to either change the label to "Copied" when a user clicks the button or simply remove the ternary operator: `label={t<string>('Copy')}`

Reply:
changes the name Copy to Copied when address is copied